espnet2/speechlm/core_lm/ar.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+
+# Copyright 2024 Jinchuan Tian
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+# Implementation of naive langauge model with codec interleave
+
+from typing import Tuple, Dict
+import torch
+import logging
+
+from espnet2.speechlm.core_lm.abs_core_lm import AbsCoreLM, SpeechLMInferenceOptions
+from espnet2.speechlm.module.module import (
+    TransformerLayer,
+    PositionalEncoding,
+)
+from espnet2.speechlm.net_utils import (
+    ce_loss,
```

Comment:
quick question, is ce_loss a better option in the task compared to label-smoothing ce loss?

Reply:
Not sure if label-smooth is needed. We can add an argument to add the label smooth factor if needed.